src/components/AssetsTable/index.jsx

```diff
@@ -301,7 +304,6 @@ export class AssetsTable extends React.Component {
       deletedAsset,
       elementToFocusOnModalClose: this.statusAlertRef,
       modalOpen: false,
-      statusAlertOpen: true,
     });
```

Comment:
Looks like statusAlertOpen in the state was removed in favor of the ternary on line 414 -- out of curiosity can you explain the logic that went through that decision?

Reply:
I didn't think it was necessary as the state reflects whether the status alert is rendered. Does that seem like the right approach?

Reply:
I like the simplicity, but we'll need to check if dynamically adding an element with the alert role is accessible or if it needs to be on the page at DOM load to be registered by screen readers.

Reply:
Good catch @thallada

Reply:
It sounds like we actually *do* need to keep the alert element in the page, right? I can update the rendering for this case.

Reply:
Yep, I checked with Mark and he said that, while some browsers and screen readers are okay with dynamically adding it, we should keep the alert element on the page for compatibility with most screen readers.